core/proto/signature/gen.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
we should probably migrate them all to gen.go at some point, but it's not high priority. 
Thanks for noticing!